File: shared/src/main/java/com/holidayplanner/shared/kafka/payload/BookingRequestedPayload.java
package com.holidayplanner.shared.kafka.payload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class BookingRequestedPayload {
    private UUID bookingId;
    private UUID familyMemberId;
    private UUID eventTermId;
    private String status;
    private String parentEmail;
}

File: BookingService/BookingService/DTO/payloads/BookingRequestedPayload.cs
namespace BookingService.DTO.payloads;

public class BookingRequestedPayload
{
    public Guid BookingId { get; set; }
    public Guid FamilyMemberId { get; set; }
    public Guid EventTermId { get; set; }
    public string? Status { get; set; }
    public string? ParentEmail { get; set; }
}
